client/pages/jobs/[company-name]/[jobTitle].tsx

```diff
@@ -0,0 +1,5 @@
+import JobDetail from '../../../components/jobs/JobDetail';
+
+export default function JobDetails() {
```

Comment:
What is the purpose of `JobDetails` here? What are we doing with this component?

Reply:
It's just for the single export function for the page to have.